lib/lightning_web/components/new_inputs.ex

```diff
@@ -367,6 +372,69 @@ defmodule LightningWeb.Components.NewInputs do
     """
   end
 
+  def input(%{type: "tag"} = assigns) do
+    assigns =
+      assign_new(assigns, :tags, fn -> parse_tags_value(assigns[:value]) end)
+
+    ~H"""
+    <div phx-feedback-for={@name} class="tag-input-container">
+      <.label :if={@label} for={@id} class="mb-2">
+        {@label}<span :if={Map.get(@rest, :required, false)} class="text-red-500"> *</span>
+      </.label>
+
+      <div class="relative">
+        <input
+          type="text"
+          name={@name}
+          id={@id}
+          phx-hook="TagInput"
+          placeholder={@placeholder}
+          class={[
+            "focus:outline focus:outline-2 focus:outline-offset-1 block w-full rounded-lg text-slate-900 focus:ring-0 sm:text-sm sm:leading-6",
+            "phx-no-feedback:border-slate-300 phx-no-feedback:focus:border-slate-400 disabled:cursor-not-allowed disabled:bg-gray-50 disabled:text-gray-500",
+            @errors == [] &&
+              "border-slate-300 focus:border-slate-400 focus:outline-indigo-600",
+            @errors != [] &&
+              "border-danger-400 focus:border-danger-400 focus:outline-danger-400",
+            @class
+          ]}
+          {@rest}
+        />
+        <input type="hidden" name={@name} id={@id} value={serialize_tags(@tags)} />
```

Comment:
🤔 same `name` and `id` ? How is it still sending in an array then? 
Also, I'm getting this error when I open the publish workflow template
<img width="1435" alt="Screenshot 2025-04-28 at 10 05 23" src="https://github.com/user-attachments/assets/0b0c3037-7c84-4108-a2db-c6a5d1fb82f6" />

Reply:
This is now fixed @midigofrank !